src/components/tree/phyloTree/renderers.js

```diff
@@ -103,6 +103,7 @@ export const drawTips = async function drawTips() {
 
   for (let i = 0, j = 0; i < this.nodes.length; i += BATCH_SIZE, j++) {
     this.groups.tips[j] = this.groups.tips[j] || this.groups.tips.append("g").attr("id", `tips-${i}`);
+    this.groups.tips[j].selectAll(".tip").remove();
```

Comment:
By adding this line here it forces a redraw at the end. Should not impact initial draw significantly but should ensure that the final state of the animation is coherent. Also similar to what happens on master now (if one quickly switches between layouts, the animation "jumps" at the end)